src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/AosPackPrinter.java

```diff
@@ -61,9 +62,49 @@ public void sendAosLetterToRespondent(final CaseData caseData, final Long caseId
         }
     }
 
+    public void sendJudicialSeparationAoSLetterToRespondent(final CaseData caseData, final Long caseId) {
+
+        final List<Letter> currentAosLetters = aosLetters(caseData, NOTICE_OF_PROCEEDINGS_APP_2);
+
+        if (!isEmpty(currentAosLetters)) {
+
+            final String caseIdString = caseId.toString();
+            final Print print = new Print(currentAosLetters, caseIdString, caseIdString, LETTER_TYPE_RESPONDENT_PACK);
+
+
+            final UUID letterId = bulkPrintService.printAosRespondentPack(print, false);
+            log.info("Letter service responded with letter Id {} for case {}", letterId, caseId);
+        } else {
+            log.warn(
+                "AoS Pack print for respondent has missing documents. Expected documents with type {} , for Case ID: {}",
```

Comment:
Personal note: We need to investigate adding an exception being thrown here instead of just a warning, to alert devs / caseworkers that letters are missing so we aren't going to send the applicant their letters